src/Microsoft.ML.Data/StaticPipe/PipelineColumn.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Data.StaticPipe.Runtime
```

Comment:
1. Is the idea that the `Runtime` namespace is separate from the rest of the `StaticPipe` types because an end user isn't supposed to use types in this namespace?  (Similar to how the `Microsoft.ML.Runtime` namespace isn't supposed to contain "user facing" types?)
2. In the .NET BCL team, we physically lay out our .cs files to match the namespace they are in. It makes the code easier to understand and navigate. Can the classes that are in the `Runtime` namespace be moved to a sub folder named `Runtime`?